config/delta-producer/submissions/export.json

```diff
@@ -0,0 +1,262 @@
+{
```

Comment:
Yes I know, Initially I thought that we would get them from contacthub, but it turned out to be more complicated since contacthub has a better model, i.e. got rid of the ext: stuff, so we'll need to deal with that.                                                             
The reason to get it from there, is that we the publication graph could be smaller, so less load. 

So I would suggest to start with an static dump of this information, and that we gradually change and ingest from contachub later